docs/User Manual.rst

```diff
@@ -0,0 +1,1014 @@
+      <name1> { id <idNumber1>; type <universeType>; *keywords* } 
+      <name2> { id <idNumber2>; type <universeType>; *keywords* } 
+      ...
+      <nameN> { id <idNumberN>; type <universeType>; *keywords* } 
+      }
+      
+Several ``universeTypes`` are possible:
+
+* cellUniverse, composed of the union of different cells. Note that overlaps are forbidden, but there is no check to find overlaps
+
+  - cells: array containing the ``cellIds`` as used in the cell definition
+  - origin (*optional*, default = (0.0 0.0 0.0)): (x y z) array with the origin of the universe. [cm]
+  - rotation (*optional*, default = (0.0 0.0 0.0)): (x y z) array with the rotation angles in degrees applied to the universe. [°]
+
+Example: ::
+
+      uni3 { id 3; type cellUniverse; cells (1 2 55); origin (1.0 0.0 0.0); rotation (0.0 90.0 180.0); }
+
+* pinUniverse, composed of infinite co-centred cylinders
+
+  - radii: array containing the radii of the co-centred cylinders. There must be an entry equal to 0.0, which corresponds to the outermost layer, which is infinite. [cm]
+  - fills: array containing the names or ids of what is inside each cylindrical shell. The order of the fills must correspond to the order of the corresponding radii. An entry can be a material name, the keyword ``void``, or a   ``u<id>``, where ``id`` is the id of a defined universe
+  - origin (*optional*, default = (0.0 0.0 0.0)): (x y z) array with the origin of the universe. [cm]
+  - rotation (*optional*, default = (0.0 0.0 0.0)): (x y z) array with the rotation angles in degrees applied to the universe. [°]
+
+Example: ::
+
+      uni3 { id 3; type pinlUniverse; radii (0.2 1.0 1.1 1.3 0.0); fills (u<1> fuel void clad coolant); }
+
+* latUniverse, cartesian lattice of constant pitch
+
+  - shape: (x y z) array of integers, stating the numbers of x, y and z elements of the lattice. For a 2D lattice, one of the entries has to be 0
+  - pitch: (x y z) array with the x, y and z lattice pitches. In a 2D lattice, the value entered in the third dimension is not used. [cm]
+  - padmat: material name or universe index (u<id>) that fills the possible extra space between the lattice and its bounding surface. Also the keyword ``void`` is allowed
+  - map: map that includes the universe ids of the elements of the lattice. The order is: increasing x, increasing y and then increasing z
+  - origin (*optional*, default = (0.0 0.0 0.0)): (x y z) array with the origin of the universe. [cm]
+  - rotation (*optional*, default = (0.0 0.0 0.0)): (x y z) array with the rotation angles in degrees applied to the universe. [°]
+
+Example: ::
+
+      uni_lattice { id 10; type latUniverse; shape (3 2 2); pitch (1.0 1.0 1.5); padMat u<3>; map (
+      1 2 3 // x: 1-3, y: 1, z: 1
+      4 5 6 // x: 1-3, y: 2, z: 1
+      7 8 9 // x: 1-3, y: 1, z: 2
+      10 11 12 ) } // x: 1-3, y: 2, z: 2
+
+* rootUniverse: top level universe of geometry  **NB THE DOCUMENTATION IN THE FILE IS WRONG (sample input)**
```

Comment:
You mean the missing `type` keyword in the sample dictionary? 
Maybe we could correct it in this PR as well? 

Thanks for picking that up!